### ISSUE TITLE
Can't have a property called `content` on proxy

### ISSUE THREAD

**Issue by amk221** [emberjs/ember.js#13445]
This issue initially came to light for me while working with Ember-Data, where I had a Model, with an attribute called `content`. When rendering `{{task.content}}`, I was seeing `<example@model:task::ember1570:47>` instead of the the task's content. (Initial example: http://jsfiddle.net/amk221/tjja7933/)

Here is an example _excluding_ Ember-Data http://jsfiddle.net/amk221/o2oqjgga/ 

* With the Ember-Data example, the issue could be solved by using a serializer to change the attr name to something else.
* With the non-Ember-Data example, I could just _not_ use `content`.

Basically, unsure if this should work or not.

**Comment by Serabe** [comment]
It is the expected behaviour.

> `Ember.ObjectProxy` forwards all properties not defined by the proxy itself to a proxied `content` object.

The [`Ember.ObjectProxy` documentation](http://emberjs.com/api/classes/Ember.ObjectProxy.html) has more info about it.

**Comment by Serabe** [comment]
Closing this. If you feel that the behaviour is not expected, feel free to reopen it.
